From 5f6fcaeb0b7a319c8afd32ddb48b3515b63a6c0c Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 18 Sep 2017 17:22:43 -0700
Subject: [PATCH] md: Fix build with musl

The MIPS specific header <sgidefs.h> is not provided by musl
linux kernel headers provide <asm/sgidefs.h> which has same definitions

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>

---
 pr/include/md/_linux.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pr/include/md/_linux.cfg b/pr/include/md/_linux.cfg
index aa3af78..899826f 100644
--- a/pr/include/md/_linux.cfg
+++ b/pr/include/md/_linux.cfg
@@ -511,7 +511,7 @@
 #error "Unknown MIPS endianness."
 #endif
 
-#if _MIPS_SIM == _ABI64
+#if _MIPS_SIM == _MIPS_SIM_ABI64
 
 #define IS_64
 
